fixtures/fixtures.js

```diff
@@ -55,12 +55,15 @@ var Fixtures = {
         ]
     },
     voucherStatus: {
-        "success":
-            {"success":["RVNT12345678"], "fail":[], "invalid":[]},
-        "fail":
-            {"success":[], "fail":["FAL11111111"], "invalid":[]},
-        "invalid":
-            {"success":[], "fail":[], "invalid":["INV1"]},
+        "success": [{"message": 'Voucher is valid'}],
+        "warning": [
+            {"warning": "You have already submitted voucher code XXX00000."},
+            {"warning": "It looks like the code (:code) has been used already, please double check and try again. "
+                + "If you are still unable to add the voucher code, don't worry - mark it as \"unrecorded\", "
+                + "send it in with your other vouchers and you will still be paid when we receive it."
+            }
```

Comment:
Can we change "unrecorded" to "already used" - this is what kat has written in her instructions which we'll be distributing to the traders, so it should match. Also I think "already used" will make more sense to traders.